bin/setup

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+
```

Comment:
it might be good to set "set -u" at the top so undefined var references will cause a crash -- generally good shell scripting practice.  In addition "set -e" would ensure that command failures later on will crash the script as a default.